pages/common/pio-test.md

```diff
@@ -0,0 +1,28 @@
+# pio test
+
+> Run local tests on a PlatformIO project.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/cmd_test.html>.
+
+- Run all tests in all environments of a PlatformIO project:
+
+`pio test`
+
+- Test only specific environments:
+
+`pio test --environment {{environment1}} --environment {{environment2}}`
+
+- Run only tests whose name match a specific pattern:
+
+`pio test --filter "{{pattern}}"`
+
+- Ignore tests whose name match a specific pattern:
+
+`pio test --ignore "{{pattern}}"`
+
+- Specify a port for firmware uploading:
+
+`pio test --upload-port {{upload_port}}`
```

Comment:
What precisely is mean  by a port here? I assume a COM port on Windows, but how does this work on Linux where we call them devices (e.g. `/dev/ttyUSB0`)?

Reply:
For me, it's `/dev/ttyUSB0` or `/dev/ttyACM0`, depending on the device. I just wasn't sure if this is general enough to use in in the example.

Reply:
Ah, ok. Looks like an example of program-specific language here - so let's leave it as-is.